components/camel-coap/src/test/java/org/apache/camel/coap/CoAPComponentTLSTestBase.java

```diff
@@ -419,9 +432,10 @@ private void registerTLSConfiguration(CamelContext context) throws GeneralSecuri
 
         context.getRegistry().bind("privateKey", privateKey);
         context.getRegistry().bind("publicKey", publicKey);
-        context.getRegistry().bind("trustedRpkStore", trustedRpkStore);
-        context.getRegistry().bind("failedTrustedRpkStore", failedTrustedRpkStore);
-        context.getRegistry().bind("pskStore", pskStore);
+
+        context.getRegistry().bind("advancedCertificateVerifier", advancedCertificateVerifier);
+        //        context.getRegistry().bind("failedAdvancedCertificateVerifier", failedAdvancedCertificateVerifier);
```

Comment:
commented out code to remove ro to readd correctly, I would expect some tests to fail without it

Reply:
Thanks, should have caught these!